internal/compiler/object_tree.rs

```diff
@@ -2058,15 +2067,15 @@ pub fn visit_all_named_references(
                     vis(&mut p.x);
                     vis(&mut p.y);
                 });
+                compo
```

Comment:
I wonder if this still still needed. We're in `visit_all_named_references`, and `visit_all_named_references_in_element` is called a few lines further up. `visit_all_named_references_in_element` calls `visit_element_expressions`, which this PR also fixes to visit the init_code.